### PR TITLE
Show schedule hours

### LINKED ISSUE #557: total hours counter
feature that adds a counter for cumulative hours generated in the schedule somewhere in the UI

### PR DESCRIPTION
**Pull request by ryan-conn**
## Description
Shows hours for schedules both below the schedule and in schedule details

## Screenshots
|Before (below schedule) | After |
|--|--|
| image | image |

| Before (details) | After |
|-|-|
| image | image |

## Related tasks
Closes #557.

### CODE REVIEW

File: autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleDetails/ScheduleDetails.css [hunk 1]
```diff
@@ -42,17 +47,43 @@
...
+.crn-hours-container {
+    justify-self: right;
+    display: flex;
+    align-items: center;
+}
+
+.crn-hours-container > :first-child {
+    padding-right: 8px;
+}
+
```
Comment (gannonprudhomme) [inline autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleDetails/ScheduleDetails.css:59]: Pretty sure these are unused?

File: autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleDetails/ScheduleDetails.css [hunk 2]
```diff
@@ -42,17 +47,43 @@
...
+    justify-self: center;
+}
+
+.left-padding {
+  padding-left: 8px;
+}
+
+.right-padding {
+  padding-right: 8px;
+}
```
Comment (gannonprudhomme) [inline autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleDetails/ScheduleDetails.css:79]: Seems unused

**Review by gannonprudhomme — CHANGES_REQUESTED** [review]
A few unused css classes but code wise this looks good.

Some visual comments that we can discuss in the meeting:
- Instructor text wrapping looks sort of funny
- Not sure if I want meeting days to be centered or not

**Comment by ryan-conn** [comment]
The most recent set of commits fixes the unused CSS classes and improves the grid layout of schedule details in a couple ways:
- Instructional method icons are now included in the grid to make spacing more consistent and make it possible to use a single unified grid instead of one per section
- Instructor names will overflow with ellipsis instead of wrapping and causing layout issues, see the following examples (screenshots are at 1024x768 which is lower than pretty much anyone will ever use to demonstrate how little wrapping is actually needed, ignore the scrollbar as it's a devtools rendering issue):
image
image

**Review by gannonprudhomme — APPROVED** [review]
Great solution to the wrapping problem! Works much better than allowing two lines

**Pull request merged**